components/content-service-api/typescript/src/blobs_pb.d.ts

```diff
@@ -1,9 +1,3 @@
-/**
- * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
- * Licensed under the GNU Affero General Public License (AGPL).
- * See License-AGPL.txt in the project root for license information.
- */
-
```

Comment:
I know, it's quite annoying that the generate script removes the license header. Please run
```
$ leeway run components:update-license-header
```
to add them again. Probably we should add this command to the generate script?